bootstrap.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/sh
 
-# ensure m4 dir exists 
+# if running from `meson dist`, make sure we run from the dist root
+if [ -n "$MESON_PROJECT_DIST_ROOT" ]; then
+    cd "$MESON_PROJECT_DIST_ROOT"
+# meson < 0.58
```

Comment:
I think we can just bump our meson version requirement (I seem to be on 1.2.3 myself).

Reply:
That's an option, but it does mean Debian Oldstable and Ubuntu 20.04 LTS users, for example, would not be able to use the version of meson installed via `apt` and would have to manually acquire an updated meson.

I don't like to assume what range of userbase a project wishes to support. I'll go with whatever makes you happy. (And it was easy for me to do, anyway.) :)

Reply:
Oops, missed that last comment... no point changing it now any more! We'll just get it whenever we next look at this.